addons/cookoff/script_component.hpp

```diff
@@ -29,6 +29,9 @@
 #define MAX_TIME_BETWEEN_AMMO_DET 25
 #define MAX_COOKOFF_INTENSITY 10
 
+#define COOKOFF_FLAME_STAGE 1 // How many flame executions to allow
+#define COOKOFF_FINALE_STAGE 2 // When to trigger finale stage
+
```

Comment:
This is just me, but I would like some variance here `#define COOKOFF_FLAME_STAGE (1 + floor 3)`
Purely just because I think it's fun for people to walk up to a vehicle they think is safe and have it explode in their face.
It does complicate the code a bit, and isn't blocking

Reply:
I think this should be a setting then because in my use case, I absolutely do not want more than one flameout stage